doc/classes/Viewport.xml

```diff
@@ -33,6 +33,18 @@
 				Returns the first valid [World3D] for this viewport, searching the [member world_3d] property of itself and any Viewport ancestor.
 			</description>
 		</method>
+		<method name="get_audio_listener_2d" qualifiers="const">
+			<return type="AudioListener2D" />
+			<description>
+				Returns the currently active 2D audio listener. Returns null if there are no active 2D audio listeners.
```

Comment:
I see that these were copied from `get_camera_2d` and `get_camera_3d`. Those ones make me a bit mad, but I digress. For now fix it here.
```suggestion
				Returns the currently active 2D audio listener. Returns [code]null[/code] if there are no active 2D audio listeners.
```